src/Evolution/Systems/Cce/Events/ObserveTimeStep.hpp

```diff
@@ -0,0 +1,151 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cmath>
+#include <cstddef>
+#include <pup.h>
+#include <pup_stl.h>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "IO/Observer/ObserverComponent.hpp"
+#include "IO/Observer/ReductionActions.hpp"
+#include "Options/String.hpp"
+#include "Parallel/ArrayIndex.hpp"
+#include "Parallel/GlobalCache.hpp"
+#include "Parallel/Invoke.hpp"
+#include "Parallel/Printf.hpp"
+#include "ParallelAlgorithms/EventsAndTriggers/Event.hpp"
+#include "Time/Time.hpp"
+#include "Utilities/Serialization/CharmPupable.hpp"
+#include "Utilities/System/ParallelInfo.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Tags {
+struct TimeStep;
+}  // namespace Tags
+/// \endcond
+
+namespace Cce::Events {
+
+/*!
+ * \brief %Observe the size of the time steps on the characteristic evolution.
+ *
+ * Writes reduction quantities:
+ * - `%Time`
+ * - `Time Step`
+ *
+ * The subfile will be written into the `/Cce` subgroup.
+ *
+ * All values are reported as positive numbers, even for backwards
+ * evolutions.
```

Comment:
Does backwards CCE make sense?

Reply:
I don't think so? Considering these are future null slices. Must have been left over when I copied the DG event. I'll get rid of it.